CMP1903_A1_2324/Die.cs

```diff
@@ -13,11 +13,20 @@ internal class Die
          * and one method that rolls the die, returns and integer and takes no parameters.
          */
 
-        //Property
-
+        /// <summary>
+        /// The "Die" class is responsible for defining the Die and assigning it a random value within the range 1-6. 
+        /// this is done with the help of the "Roll" method which generates the die values within the expected range.
+        /// </summary>
 
-        //Method
 
+        //Property
+        private int _dieCurVal; //Private variable created to hold the current value of the die.
```

Comment:
Potentially add a Getter for the _dieCurVal so you can get the current value stored inside of it from outside the class.